scripts/testnet/sovereignBridge/config/helper.cfg

```diff
@@ -0,0 +1,24 @@
+CONFIG_FILE=$SCRIPT_PATH/config/configs.cfg
+
+manualUpdateConfigFile() {
```

Comment:
There is no problem with the namings. There were copied from another repo and sound good because each function does something different.

In this context source is like an update of the config values hence the function name...

Reply:
I understand these were copy-pasted, but they need to follow some naming convention if they are all put into one place.

Reply:
made them camelCase